### ISSUE TITLE
channel points -> unlock themesong play (or just pay for themesong play)

### ISSUE THREAD

**Issue by tjdevries** [tjdevries/subd#11]

**Comment by davidbegin** [comment]
Yeah I think some concept of channel-points to song-access is interesting.
Here's a couple example:

- Buy sound to play once, right when you buy
  - good for sounds you hate (like sonicdrown for me) or ones that are really long (snorlax)
  
  - Buy the sound for the duration of the stream
  
- Buy the sound permanently
   - this would have to be stored in the DB
